src/extlibs/enlsvg/enlsvg_algorithm.h

```diff
@@ -5,11 +5,19 @@
 #include <limits>
 #include <vector>
 
+<<<<<<< HEAD:src/extlibs/enlsvg/ENLSVGAlgorithm.h
```

Comment:
you've got a conflict here, basically an earlier PR renamed these files, so git's a bit confused on which set to use